src/pymail

```diff
@@ -84,72 +103,91 @@ def email_recv(use_sys_argv):
 
     # Get the latest email id
     result, data = mail.search(None, "ALL")
-    msg_ids = data[0]
-    msg_id_list = msg_ids.split(' ')
+    msg_id_list = data[0].split(' ')
```

Comment:
No reason to declare two variables to do one thing if you're only going to use the var once